client/stylesheets/_toCat.scss

```diff
@@ -3709,6 +3712,10 @@ On click the images will be displayed at normal size to complete the effect
     color: #fff !important;
 } */
 
+.profileUserButtonUl > li {
+    display: inline-block !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces